### PR TITLE
Always showing first 3 other event times on event page.

### PR DESCRIPTION
**Pull request by nikomakela**
TH-1081.
image

### CODE REVIEW

**Comment by nikomakela** [comment]
The acceptance test fails on v 1.0, but the feature seems to work as expected:

image

image

**Comment by nikomakela** [comment]
> The acceptance test fails on v 1.0, but the feature seems to work as expected:
> 
> image
> 
> image

Heard from @nenonja that this test just fails from time to time and it's related to duplicated events showing up during the test..

**Review by Sartonon — COMMENTED** [review]
Looing pretty good. But I think we could easily render event list component one time and just pass different amount of events.

Also "Show more" and "Piilota" might need little bit more spacing between the list:

Screenshot 2021-04-01 at 10 59 06

File: src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx [hunk 1]
```diff
@@ -62,12 +55,29 @@ const OtherEventTimes: React.FC<Props> = ({
     return null;
   }
 
+  const shownEvents = events.slice(0, 3);
```
[the remark this answers is not in the document]
Comment (nikomakela) [reply src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx]: Yep, much better like that!

[the remark this answers is not in the document]
Comment (nikomakela) [reply src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx]: > Also "Show more" and "Piilota" might need little bit more spacing between the list

I would say that all those links needs some spacing, but this time I fixed only that one. I used `spacing-xs`.

[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx]: I actually tested margin with `spacing-xs` previously and it looked good :)

**Review by Sartonon — APPROVED** [review]
GREAT

**Pull request merged**